Makefile

```diff
@@ -8,7 +8,8 @@ generate-openapi-client:
 			-g go \
 			-i internal/spec/openapi.json \
 			-o internal/openapi-generator \
-			-c internal/spec/config.yaml
+			-c internal/spec/config.yaml \
+			--type-mappings=int64=int64
```

Comment:
I'd like this to be in https://github.com/cockroachdb/cockroach-cloud-sdk-go/blob/main/internal/spec/config.yaml instead if possible.

Reply:
Updated.